benchmarkcommon/src/test/java/com/ss/benchmark/httpclient/HttpClient.java

```diff
@@ -0,0 +1,22 @@
+package com.ss.benchmark.httpclient;
+
+import java.util.concurrent.CompletableFuture;
+
+public interface HttpClient {
+
+    //All times are milliseconds unless otherwise noted
+    static final int MAX_CONNECTION_POOL_SIZE = 200;
+    static final int CONNECT_TIMEOUT = 500;
+    static final int READ_TIMEOUT = 2_000;
+
+    void createClient(String baseUrl);
+
+    String blockingGET(String uri);
+
+    String blockingPOST(String uri, String body);
+
+    CompletableFuture<String> nonblockingGET(String uri);
+
+    CompletableFuture<String> nonblockingPOST(String uri, String body);
+
+}
```

Comment:
Would it simplify things to have this interface defined in the src tree (src/main/java)?

Reply:
Yes, I hope to make this change if @TexanHogman does not:)